Cython/Includes/cpython/array.pxd

```diff
@@ -99,7 +99,10 @@ cdef extern from *:  # Hard-coded utility code hack.
         cdef:
             Py_ssize_t ob_size
             arraydescr* ob_descr    # struct arraydescr *ob_descr;
-            __data_union data
+
```

Comment:
... then my change to the name for `__data_union` would involve changing

```
    ctypedef union __data_union "__Pyx_data_union":
```

to keep the same name in Cython in case anyone's relying on it.

But I can't add it as a suggestion because it's too far up.